### PR TITLE
Add new service calls for Squeezebox

### PR DESCRIPTION
**Pull request by rajlaud**
reflect changes in Squeezebox add query and sync pull request (#31748)

## Proposed change

Added descriptions of new services, 'call_query', 'sync', and 'unsync'.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/home-assistant/pull/31748
- This PR fixes or closes issue:

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/squeezebox.markdown [hunk 1]
```diff
@@ -79,3 +79,37 @@ For example, to play an album from your collection, create an IFTT applet like t
 - Action: JSON post query with such JSON body:
 `{ "entity_id": "media_player.squeezebox_radio", "command": "playlist", "parameters": ["loadtracks", "album.titlesearch={{TextField}}"] }`
 This can work with title search and basically any thing. The same wouldn't have worked by calling directly Squeezebox server as IFTT cannot escape the text field.
+
+### Service `call_query`
+
+Call a custom Squeezebox JSONRPC API. The results of the query will be stored in the 'query_result' attribute of the player.
```
Comment (frenck) [inline source/_integrations/squeezebox.markdown]:
```suggestion
Call a custom Squeezebox JSON-RPC API. The result of the query will be stored in the 'query_result' attribute of the player.
```

File: source/_integrations/squeezebox.markdown [hunk 2]
```diff
@@ -79,3 +79,37 @@ For example, to play an album from your collection, create an IFTT applet like t
...
+
+See documentation for this interface on `http://HOST:PORT/html/docs/cli-api.html?player=` where HOST and PORT are the host name and port for your Logitech Media Server.
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id` | no | Name(s) of the Squeezebox entities where to run the API method.
+| `command` | no | Command to pass to Logitech Media Server (p0 in the CLI documentation).
+| `parameters` | yes | Array of additional parameters to pass to Logitech Media Server (p1, ..., pN in the CLI documentation).
+
+This service can be used to integrate a Squeezebox query into an automation. For example, in a python script, you can get a list of albums available by an artist like this:
```
Comment (frenck) [inline source/_integrations/squeezebox.markdown]:
```suggestion
This service can be used to integrate a Squeezebox query into an automation. For example, in a Python script, you can get a list of albums available by an artist like this:
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @rajlaud!

Just 2 small comments. Could you please take a look? Thanks! 👍

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by rajlaud** [comment]
The underlying pull request has been approved. https://github.com/home-assistant/core/pull/31748.

**Review by frenck — APPROVED** [review]

**Pull request merged**